test/db/cmd/cmd_interpret

```diff
@@ -30,3 +30,30 @@ Eron
 
 EOF
 RUN
+
+NAME=$foo=#!pipe
+FILE=-
+CMDS=<<EOF
+#!pipe echo hello
+$foo=#!pipe echo hello
+$foo
+?e --
+"$foo=#!pipe echo hello"
+$foo
+?e --
+$-*
+e cfg.newshell=true
+$foo=\#!pipe echo hello
```

Comment:
ops, wait. I need to check one thing. There is something wrong here with newshell.

Reply:
https://github.com/radareorg/radare2/issues/17351

Reply:
that said, I think this test is ok. After all, an alias is just a string, so the only way to pass `#` is to either escape it or wrap it. For now, escaping is the only supported option though.